components/builder-api/doc/api.raml

```diff
@@ -878,6 +878,19 @@ securitySchemes:
                       description: Cannot transfer to self or non-members
                     500:
                       description: Internal server error
+        /depart:
+          post:
+            description: Leave an origin you no longer wish to be a member of
+            securedBy: [oauth_2_0]
```

Comment:
Side note - these calls are secured by bearer token, not oauth ... that should be a raml-wide update (separate PR)

Reply:
https://github.com/habitat-sh/builder/issues/1294